ambari: Add jdeb support

nareshgbhat opened the pull request:
This patch add jdeb (debian) package creation support for the following
packages.
-ambari-funtest
-ambari-logsearch
-ambari-metrics-grafana
-ambari-metrics-host-aggregator

Signed-off-by: Naresh Bhat <naresh.bhat@linaro.org>

## What changes were proposed in this pull request?

jdeb support for branch 2.7

## How was this patch tested?

I did a build on AArch64 machine with Debian 9 stretch

Please review [Ambari Contributing Guide](https://cwiki.apache.org/confluence/display/AMBARI/How+to+Contribute) before opening a pull request.

asfgit commented:
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/Ambari-Github-PullRequest-Builder/3229/
Test PASSed.

nareshgbhat committed:
AMBARI-23903: Add jdeb support

This patch add jdeb (debian) package creation support for the following
packages.
-ambari-funtest
-ambari-infra
-ambari-logsearch
-ambari-metrics-grafana
-ambari-metrics-host-aggregator

Signed-off-by: Naresh Bhat <naresh.bhat@linaro.org>

nareshgbhat commented:
@oleewere I have used version 1.4 for infra and logsearch on v2.7 branch.  Thanks for suggestions.

asfgit commented:
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/Ambari-Github-PullRequest-Builder/3623/
Test PASSed.

oleewere reviewed (approved):
lgtm
